z_infra_provisioning/cloud_infra_center/ocp_upi/README.md

```diff
@@ -80,53 +81,123 @@ Depending on the type of image registry backend an additional 100 GB volume.
   - it will host two records: for API and apps access
   - inventory: `vars.os_dns_domain`
 - Bastion node
-  - If you don't have your own DNS server and Load Balancer, you can choose to configure them on a RHEL8 machine, which called a "Bastion Node".
-  - If using bastion node, ensure that you can ssh to bastion node via root user without password.
-  - If using bastion node, uncomment `bastion` and `vars.cluster_nodes` part in inventory.yaml and fill the information of bastion node and OpenShift cluster.
-  - inventory: `bastion`, `vars.cluster_domain_name`, `vars.cluster_nodes`
+  - Ensure network connectivity among CIC control nodes, DNS, DHCP(if used),HAProxy, Installer or Bastion node.
```

Comment:
I don't think we should refer "DHCP" here.

Reply:
Yes, I suppose so, just remove it.